packages/loader/container-loader/src/deltaManager.ts

```diff
@@ -1167,12 +1165,11 @@ export class DeltaManager
         this.disconnectFromDeltaStream(error.message);
 
         // If reconnection is not an option, close the DeltaManager
-        const canRetry = canRetryOnError(error);
-        if (this.reconnectMode === ReconnectMode.Never || !canRetry) {
+        if (this.reconnectMode === ReconnectMode.Never || !error.canRetry) {
             // Do not raise container error if we are closing just because we lost connection.
             // Those errors (like IdleDisconnect) would show up in telemetry dashboards and
```

Comment:
>o [](http://example.com/codeflow?start=58&length=1)

I think we should keep using the common utility api to see if the error is retriable or not. That way if in future we want to change defn of retriable then we can do that at 1 place rather than having to find canRetry at different places.

Reply:
Ok, sure. I'm a sucker for type-safety, that's what drew me to make the change :P